source/stylesheets/refills/_animate.scss

```diff
@@ -0,0 +1,87 @@
+$base-accent-color: #477dca !default;
+
+.animated {
+  -webkit-animation-duration: 1s;
+          animation-duration: 1s;
+  -webkit-animation-fill-mode: both;
+          animation-fill-mode: both;
+}
+
+.animated.infinite {
+  -webkit-animation-iteration-count: infinite;
+          animation-iteration-count: infinite;
+}
+
+.animated.alternate {
+  -webkit-animation-direction: alternate;
+          animation-direction: alternate;
+}
+
+.animated.iteration {
+  -webkit-animation-iteration-count: 2;
+          animation-iteration-count: 2;
+}
+
+//  Tweak the keyframes below. Or swap them with 
+//  another animation pattern at 
+//  https://github.com/daneden/animate.css/tree/master/source
+//
+/////////////////////////////////////////////////////////////
```

Comment:
I still feel strong about not having these types of comments…they really dirty things up.